### DISCUSSION TITLE
[Q-TOUCH 2ch wall switch] inappropriate behavior of the LEDs status

### DISCUSSION THREAD

**Discussion by JackGie** [arendst/Tasmota#16541] [Q&A] [answered]
Hi everyone,
 I have a Q-TOUCH 2CH wall switch as in the link below. 
[https://electro24.pl/wlacznik-dotykowy-swiatla-szklany-pilot-WIFI](url)

I checked the operation on the original software (Ewelink) - everything works ok.
 After switching to Tasmota, I have a little problem with the status LEDs. 
Blue color means - relay OFF
Red means - relay ON 

If  switch 1 / relay 1 goes ON -> Led1 changes from blue to red, **BUT** Led2 goes out (should remain unchanged – BLUE). Same behav at switch2.
Why 2nd led goes out after toggeling the first one (from OFF to ON) ?

Standby mode (relays OFF)
•	If relay1 is OFF, led1 is BLUE.
•	If relay 2 is OFF, led2 is BLUE 
Relay 1 ON
•	If relay1 is ON, led1 is RED.
•	If relay 2 is OFF, led2 is OUT
Relay 2 ON
•	If relay1 is OFF, led1 is OUT.
•	If relay 2 is ON, led2 is RED
Relay 1&2 ON
•	If relay1 is ON, led1 is RED.
•	If relay 2 is ON, led2 is RED

Please help

**Comment by barbudor** [comment]
We can't help without details on how you configured Tasmota
I can't see any page for that device in https://templates.blakadder.com/

**Reply by JackGie** [reply]
Thank you for your response

The switch is Q-TOUCH 2ch, ESP8285 sits on board, on triacs (silent switching). Uploaded Tasmota 12.1.1

Settings:
- Module type: Sonoff T1 2ch (29)
- Template: based by Sonoff T1 2ch (29)

The Sonoff T1 setting is the only one that worked, but there is a problem with the LEDs going out.

BTW.  there is template to 1ch switch version of QTOUCH on repo
[https://templates.blakadder.com/q-touch_QIP_W1_WIFI_A-TOUCH.html](url)

**Reply by barbudor** [reply]
Don't use the Sonoff T1 module, it is meant for the T1 only. 

As the device is unknown in the template database, you have to figure yourself which GPIO are connected to components on your device
https://tasmota.github.io/docs/Configuration-Procedure-for-New-Devices/

**Reply by JackGie** [reply]
Yes, I know. I ran the entire procedure early. The problem is that, as I wrote above, the executive element here is a triac (you cannot hear clicking) and you do not know what to assign as a relay and what as a led.

Anyway, it looked like this for me:
STEP1 ID RELAYS_1
1 GPIO_4 - none
2 GPIO_5 - Relay_2
3 GPIO_12 - Relay_1
4 GPIO_13 - all LEDs goes out
5 GPIO_14 - none
6 GPIO_15 - none
7 GPIO_16 - none


STEP2 ID RELAYS_2 (nothing happens, the panel does not respond, it is dead - the LEDs are off)

1 GPIO_0 - none
2 GPIO_1 - none
3 GPIO_2 - none
4 GPIO_3 - none

STEP3 ID RELAYS_3  (nothing happens, the panel does not respond, it is dead - the LEDs are off)

1 GPIO_9 - none
2 GPIO_10 - none

**Comment by JackGie** [comment, marked as answer]
Okay, I finally had some time ...

After studying the documentation and trying few different settings, I finally have a SOLUTION

Working config (template)
`{"NAME":"QTOUCH_2G","GPIO":[32,1,1,1,0,225,33,0,224,320,0,0,0,0],"FLAG":0,"BASE":29}`

BUT !!!!  the blue backlight does not work in this case. 
Additionally, the **LedPower** parameter should be changed in the console

`LedPower 1`


and that's it

BTW.  I added  new template to repository
